ui/tests/unit/services/permissions-test.js

```diff
@@ -163,18 +163,26 @@ module('Unit | Service | permissions', function(hooks) {
       'sys/auth': {
         capabilities: ['deny'],
       },
-      'sys/leases/lookup': {
-        capabilities: ['read'],
+      'identity/group/id': {
+        capabilities: ['list', 'read'],
       },
     };
     service.set('exactPaths', accessPaths);
-    assert.equal(service.hasNavPermission('access', 'leases'), true);
+    assert.equal(service.hasNavPermission('access', 'groups', ['list', 'read']), true);
   });
 
   test('hasNavPermission returns false if a policy does not include access to any paths', function(assert) {
     let service = this.owner.lookup('service:permissions');
-    service.set('exactPaths', {});
-    assert.equal(service.hasNavPermission('access'), false);
+    const accessPaths = {
+      'sys/auth': {
+        capabilities: ['deny'],
+      },
+      'identity/group/id': {
+        capabilities: ['read'],
+      },
+    };
+    service.set('exactPaths', accessPaths);
+    assert.equal(service.hasNavPermission('access', 'groups', ['list', 'read']), false);
```

Comment:
Do you have a test somewhere for when you don't pass through capabilities at all (so testing the stuff I quibble about above)? I'm sure it works, just might be worthwhile adding to cope with everything and cover you for in the future if things change.

Reply:
i just added an extra assertion for this. thanks for the suggestion!